renovate.json

```diff
@@ -19,5 +19,16 @@
     }
   ],
   "platformAutomerge": true,
-  "platformCommit": true
+  "platformCommit": true,
+  "regexManagers": [
+    {
+      "fileMatch": ["(^|/)zarf.yaml$"],
+      "matchStringsStrategy": "recursive",
+      "matchStrings": [
+        "charts:(.|\n)*?((^\\s{4}[\\w:]+)|\n$|^\\s{2}-)",
```

Comment:
My regexp fu is failing me today. I'm having a little trouble understanding what the goal of the second capture group is `((^\\s{4}[\\w:]+)|\n$|^\\s{2}-)`

Reply:
That will try to catch the end of the charts section (either 4 space chars then word colon (i.e. the start of a new section `    scripts:`), an empty newline (i.e. the end of the file ``), or 2 space chars and a dash (i.e. the start of a new component `  - name:`))

Reply:
(though it may not be working correctly... lemme play around a bit more)

Reply:
NVM, forgot about the double slashes from JS land 

https://regex101.com/r/XhpCdJ/1

Reply:
(just removed an uneeded group btw)